File: New folder/notes.ex
# elixir notes
defmodule Notes do
  def greet(name), do: "hello #{name}"
end

IO.puts(Notes.greet("practice"))

nums = [1, 2, 3]
doubled = Enum.map(nums, fn n -> n * 2 end)
IO.inspect(doubled)

# TODO: try pattern matching / genserver next

File: New folder/notes.dart
// dart notes
String greet(String name) => "hello $name";

void main() {
  print(greet("practice"));

  var nums = [1, 2, 3];
  var doubled = nums.map((n) => n * 2).toList();
  print(doubled);
}
// TODO: try async streams next
